setup.py

```diff
@@ -101,6 +101,6 @@
         ]
     ),
     package_data={
-        "": ["*.j2"],
+        "": ["*.j2", "**/*.j2"]
```

Comment:
Ah, sure! Only, we won't need the `"*.j2"` anymore as `"**/*.j2"` includes that (and I suspect `"*.j2"` never worked as intended :see_no_evil:).